src/filtrering/filtrering-label-container.tsx

```diff
@@ -284,4 +280,20 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
     }
 });
 
+function getLabel(singleValue: any, key: any, enhettiltak: any): string {
+    if (key === 'tiltakstyper') {
+        return enhettiltak[singleValue];
+    }
+    if (singleValue?.label) {
+        return singleValue.label;
+    }
+    if (FilterKonstanter[key] && FilterKonstanter[key][singleValue]) {
+        return FilterKonstanter[key][singleValue];
+    }
+    if (FilterKonstanter[singleValue]) {
+        return FilterKonstanter[singleValue];
+    }
+    return singleValue;
+}
+
 export default connect(null, mapDispatchToProps)(FiltreringLabelContainer);
```

Comment:
Denne funksjonen skrev Stian for at vi skulle komme oss rundt å bruke forskjellig verdi på variabelen i filter-konstanter.ts og variabelen som sendes til backend. Jeg er litt usikker på om vi skal beholde den